packages/teleport/src/Player/Player.tsx

```diff
@@ -36,23 +36,42 @@ export default function Player() {
     'recordingType',
     useLocation().search
   ) as RecordingType;
+  const durationMs = parseFloat(
+    getUrlParameter('durationMs', useLocation().search)
+  );
+
+  const validRecordingType =
+    recordingType === 'ssh' || recordingType === 'desktop';
+  const validDurationMs =
+    recordingType !== 'desktop' || // durationMs only needed for desktop recordings
+    (!isNaN(durationMs) && Number.isInteger(durationMs));
```

Comment:
i think `isNan` and `Number.isInteger` seems to be doing the same thing? also, we are `parsing for float` but then check if its an integer. couple of questions:

1) do we expect an integer? b/c there is `parseInt`
2) are we aware that parseXXX parses beginning numbers but ignores the rest ie `parseXXX("23something") = 23`? if not, i suggest taking look at `Number(string|number)`

Reply:
I'm just trying to check that it's a number with no alphabetic or other characters and no decimal place. I think `Number.isInteger(Number(durationMs))` gives me what I want.